frappe/patches/v13_0/update_list_user_settings.py

```diff
@@ -0,0 +1,23 @@
+import frappe
+import json
+from frappe.model.utils.user_settings import update_user_settings, sync_user_settings
+
+def execute():
+	""" Update list_view's sort by property from __UserSettings """
+
+	users = frappe.db.sql("select distinct(user) from `__UserSettings`", as_dict=True)
+
+	for user in users:
+		# get user_settings for each user
+		settings = frappe.db.sql("select * from `__UserSettings` \
+			where user={0}".format(frappe.db.escape(user.user)), as_dict=True)
```

Comment:
Use `frappe.qb` instead of adding another raw sql! Same for above...Although seems like even `frappe.get_all` would suffice.

What was the reason to write raw sqls?

Reply:
Done